lib/functions.rb

```diff
@@ -3,11 +3,13 @@
 
 require 'find'
 
```

Comment:
I missed something. You have to change "RUBY_MARKER_PATTERN = /scythe_probe\s*\s*\("(\w+)\"\)s*/" 
back to "RUBY_MARKER_PATTERN = /scythe_probe\s*\(\s*\"(\w+)\"\s*\)/".
Sorry.